src/idl_gen_cpp.cpp

```diff
@@ -2416,7 +2442,17 @@ class CppGenerator : public BaseGenerator {
             break;
           }
           case BASE_TYPE_BOOL: {
-            code += "_fbb.CreateVector(" + value + ")";
+            if (!NativeVectorDefined(&field)) {
```

Comment:
Both code and comments are repeated 3x below.. maybe stick it in a function?

Reply:
Done. VectorAccessor() takes care of this now.